packages/components/image/__tests__/image.test.tsx

```
@@ -26,7 +56,89 @@ describe("Image", () => {
+        set(fn) {
+          imageOnError = fn;
+          this._onerror = fn;
+        },
+        configurable: true,
+      });
+    }
+
+    trackImageOnError();
+
+    const cleanup = () => {
+      delete window.Image.prototype._onerror;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)